kfdefs/overlays/moc/smaug/opf-jupyterhub/blackbox-exporter/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Now that this kustomization exists, you can add `blackbox-exporter` in the [`opf-jupyterhub` kustomization](https://github.com/operate-first/apps/blob/20be019e462d709ccc8bc03ab1c5d26bc7a9546c/kfdefs/overlays/moc/smaug/opf-jupyterhub/kustomization.yaml#L7). So these manifests will then be deployed along with jupyterhub

Reply:
Great! I just did it